editor/src/components/canvas/controls/grid-controls-for-strategies.tsx

```diff
@@ -211,6 +217,45 @@ export function gridMeasurementHelperDataFromElement(
   }
 }
 
+function useObserversToWatch(elementPathOrPaths: Array<ElementPath> | ElementPath): number {
+  // Used to trigger extra renders.
+  const [counter, setCounter] = React.useState(0)
+  const bumpCounter = React.useCallback(() => {
+    setCounter((value) => value + 1)
+  }, [])
+
+  // Need to use the mount count for the callback trigger.
+  const mountCount = useEditorState(
+    Substores.canvas,
+    (store) => store.editor.canvas.mountCount,
+    'useObserversToWatch mountCount',
+  )
+
+  React.useEffect(() => {
+    // Add the change callback(s) for the element path or paths.
+    if (Array.isArray(elementPathOrPaths)) {
+      for (const elementPath of elementPathOrPaths) {
+        addChangeCallback(mountCount, elementPath, bumpCounter)
+      }
+    } else {
+      addChangeCallback(mountCount, elementPathOrPaths, bumpCounter)
+    }
+
+    // Cleanup to remove the callback(s) for the element path or paths.
+    return () => {
```

Comment:
I like to use this form which is more self documenting

```suggestion
    return function cleanup {
```